server/webex.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"golang.org/x/oauth2"
+
+	webexteams "github.com/jbogarin/go-cisco-webex-teams/sdk"
+	"gopkg.in/resty.v1"
+)
+
+type WebexUserInfo struct {
+	ID          string   `json:"id"`
+	Type        string   `json:"type"`
+	Emails      []string `json:"emails"`
+	DisplayName string   `json:"display_name"`
+	FirstName   string   `json:"first_name"`
+	LastName    string   `json:"last_name"`
+	Nickname    string   `json:"nickname"`
+	Avatar      string   `json:"avatar"`
+	Timezone    string   `json:"timezone"`
+
+	Roles []string `json:"roles"`
+	OrgID string   `json:"org_id"`
+}
+
+// FromWebexPerson is a helper to build out a WebexUserInfo from a webex object
+func (self *WebexUserInfo) FromWebexPerson(person *webexteams.Person) {
+	self.ID = person.ID
+	self.Emails = person.Emails
+	self.DisplayName = person.DisplayName
+	self.FirstName = person.FirstName
+	self.LastName = person.LastName
+	self.Nickname = person.NickName
+	self.Avatar = person.Avatar
+	self.Timezone = person.TimeZone
+	self.Type = person.PersonType
+
+	self.OrgID = person.OrgID
+	self.Roles = person.Roles
+}
+
+// WebexOAuthSession
+type WebexOAuthSession struct {
+	UserID string       `json:"user_id"`
+	Token  oauth2.Token `json:"token"`
+}
+
+// NewWebexClient is a helper to create a new webex sdk client
+// references:
+//   https://github.com/jbogarin/go-cisco-webex-teams
+//   https://github.com/jbogarin/go-cisco-webex-teams/blob/master/examples/people/main.go#L15-L18
+func NewWebexClient(token string) (*webexteams.Client, error) {
```

Comment:
This function will never return an error

Reply:
Updated to check for token being empty.  Oversight on my part forgetting to put the check in.